src/vorta/assets/exclusion_presets/browsers.json

```diff
@@ -0,0 +1,71 @@
+[
+    {
+        "name": "Chromium cache and config files",
+        "slug": "chromium-cache",
+        "patterns":
+        [
+            "*.config/chromium/*/Local Storage",
```

Comment:
Why did we drop the format syntax specifier? Prepending `fm:` to fnmatch patterns makes sense for those build-in strings because they will continue to work when borg changes the default pattern syntax.

Reply:
Maybe this change was added after I forked the previous PR. Will double-check and add it back.

Reply:
Also not in the [original PR](https://github.com/borgbase/vorta/pull/1742/files#diff-5276b23f1b5068ea8466dc6e08b4db034498e24e00214e62bff2b676988789d7). Let's add this here.

Reply:
I've added the matcher and made some exclusions more specific, so they will match a whole file name and not a partial one. E.g. we want to match `.config/blah` but not `blah.config`.